libraryValidations/JavaScript/README.md

```diff
@@ -0,0 +1,13 @@
+# JavaScript Feature Management Validation Tests
```

Comment:
Can you update your PR to add yourself to this script Ross added. https://github.com/microsoft/FeatureManagement/blob/main/libraryValidations/run_all_tests.ps1